packages/react/src/components/autocomplete/multi-autocomplete.tsx

```diff
@@ -154,8 +146,7 @@ export const MultiAutocomplete: FC<MultiAutocompleteProps> = ({
   )
   const {
     className,
-    isClearable = true,
-    clearable = isClearable,
+    clearable,
```

Comment:
Please add default value `true`.

Reply:
@hirotomoyamada Thanks for pointing it out! Removed it!